js/build.js

```diff
@@ -260,7 +260,7 @@
             });
 
             ed.on('focus', function() {
-              if (!hasValue) {
+              if (!widgetData.html) {
```

Comment:
`var hasValue = !!widgetData.html;` is declared at the top, which means `!hasValue` is the same as `!widgetData.html`. 

Why is this change required?

Reply:
when we click on first time on textbox the exiting placeholder text clear and that time variable `var hasValue = !!widgetData.html` is `false` - expected behavior
but when we adding any data and apply link on that text and save & close then also we got `var hasValue = !!widgetData.html;` is `false` so on focus it clear the textbox value. - unexpected behavior.

when we use the `!widgetData.html` condition then we getting the expected behavior in both case.

Reply:
👍🏻